<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1" colspan="2">
     <widget class="QTextBrowser" name="modelResponse"/>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>19</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Send message</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>User prompt:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="5">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>AI answer:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1" colspan="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>3</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QPlainTextEdit" name="userMessage"/>
    </item>
    <item row="0" column="3" rowspan="5">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="1">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionManage_agents"/>
    <addaction name="actionInterface_settings"/>
    <addaction name="actionConnection_settigns"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionhi">
   <property name="text">
    <string>hi</string>
   </property>
  </action>
  <action name="actionManage_Agents">
   <property name="text">
    <string>Manage Agents</string>
   </property>
  </action>
  <action name="actionEdit_options">
   <property name="text">
    <string>Edit options</string>
   </property>
  </action>
  <action name="actionEdit_connection">
   <property name="text">
    <string>Edit connection</string>
   </property>
  </action>
  <action name="actionManage_agents">
   <property name="text">
    <string>Manage Agents</string>
   </property>
  </action>
  <action name="actionInterface_settings">
   <property name="text">
    <string>Interface settings</string>
   </property>
  </action>
  <action name="actionConnection_settigns">
   <property name="text">
    <string>Connection settigns</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
